ref(project-upstream): Emit error on multiple project fetch failures (#2700)
getsentry/sentry-relay: CHANGELOG.md

- Add TTID and TTFD tags to mobile spans. ([#2662](https://github.com/getsentry/relay/pull/2662))
- Scrub all DB Core Data spans differently. ([#2686](https://github.com/getsentry/relay/pull/2686))
- Support generic metrics extraction version 2. ([#2692](https://github.com/getsentry/relay/pull/2692))

> - Emit error on continued project config fetch failures for an interval. ([#2700](https://github.com/getsentry/relay/pull/2700))

Reviewer: ```suggestion
- Emit error on continued project config fetch failures after a time interval. ([#2700](https://github.com/getsentry/relay/pull/2700))
```

I think this makes more sense ? 🤔
